### PR TITLE
Add additional filters to operator hub and developer catalog

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Add Provider Type and Installed filters to operator hub.
Add the Installed Operators to the developer catalog type filter.

https://jira.coreos.com/browse/MKTPLC-174
https://jira.coreos.com/browse/MKTPLC-175

UI changes are documented in https://docs.google.com/document/d/1FTvPacdQFpAdrH7qrZWmcbUAqBmcNb5BAqdbT3OOGf0/edit?ts=5c2ed2df

@serenamarie125 @tlwu2013

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/operator-hub/operator-hub-community-provider-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  render() {
+    const {show, close } = this.props;
+    const { ignoreWarnings } = this.state;
+
+    const messageText = (
+      <p>
+        These are operators which have not been vetted or verified by Red Hat.  Community Operators should be used with
+        caution because their stability is unknown.  Red Hat provides no support for Community Operators.
+        <span className="co-modal-ignore-warning__link">
+          {RH_OPERATOR_SUPPORT_POLICY_LINK && (
```
[the remark this answers is not in the document]
Comment (jeff-phillips-18) [reply frontend/public/components/operator-hub/operator-hub-community-provider-modal.tsx]: The empty span is used to separate the `Do you want..` from the rest of the text when there is no link.

**Comment by jeff-phillips-18** [comment]
@spadgett updated based on your comments

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by galletti94** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
#1063 needs to merge before anything will pass CI :/

**Comment by galletti94** [comment]
sorry

**Comment by spadgett** [comment]
No worries

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**